_data/overviews.yml

```diff
@@ -259,6 +259,11 @@
       icon: cog
       url: "compiler-options/errors.html"
       description: "A new engine for more user-friendly error messages, printing chains of dependent implicits and colored found/required type diffs."
+    - title: Optimizer
+      by: Lukas Rytz and Andrew Marki
+      icon: cog
+      url: "compiler-options/optimizer.html"
+      description: "The compiler can perform various optimizations as specified by compiler options."
```

Comment:
s/as specified by compiler options/in a configurable manner/

Reply:
> The compiler is pleased to perform such optimizations as the user may wish to configure by way of compiler settings or command line options.

Reply:
I deleted the clause as extraneous, though my previous suggestion is pretty good.